public/js/pages/localweb.js

```diff
@@ -0,0 +1,114 @@
+requirejs.config({
+  baseDir: "/js",
+  paths: {
+    "analytics": "/bower_components/webmaker-analytics/analytics",
+    "jquery": "/bower_components/jquery/jquery.min",
+    'languages': '/bower_components/webmaker-language-picker/js/languages',
+    'list': '/bower_components/listjs/dist/list.min',
+    'fuzzySearch': '/bower_components/list.fuzzysearch.js/dist/list.fuzzysearch.min',
+    'magnific-popup': '/bower_components/magnific-popup/dist/jquery.magnific-popup.min',
+    'transition': '/bower_components/bootstrap/js/transition',
+    'tabzilla': 'https://mozorg.cdn.mozilla.net/tabzilla/tabzilla',
+    'selectize': '/bower_components/selectize/dist/js/standalone/selectize.min',
+    'collapse': '/bower_components/bootstrap/js/collapse',
+    'carousel': '/bower_components/bootstrap/js/carousel'
+  },
+  shim: {
+    'transition': ['jquery'],
+    'collapse': ['jquery'],
+    'carousel': ['jquery']
+  }
+});
+
+require(["jquery", "languages", "selectize", "transition", "collapse", "carousel", "magnific-popup"], function ($, languages, selectize ) {
+  "use strict";
+  var $window = $(window);
+  //parallax
+  $(window).scroll(function() {
+    var yPos = -($(window).scrollTop() / 50);
+    $("#moi-video").css({ backgroundPosition: "50% "+ yPos + "px" });
+  });
+  // video lightbox
+  $(".moi-video-link").magnificPopup({
+    type: "iframe"
+  });
+  // quote carousel
+  $("#quote-carousel").carousel();
+  // smooth scrolling to anchors. modified from http://css-tricks.com/snippets/jquery/smooth-scrolling/
+  $(".go-to-anchor-btn").click(function() {
+    if (location.pathname.replace(/^\//,'') == this.pathname.replace(/^\//,'') && location.hostname == this.hostname) {
+      var target = $(this.hash);
+      target = target.length ? target : $('[name=' + this.hash.slice(1) +']');
+      if (target.length) {
+        $('html,body').animate({
+          scrollTop: target.offset().top
+        }, 600);
+        return false;
+      }
+    }
+  });
+  var $element = $('#langPicker');
+  var highlightLangs = [ "bn", "hi-IN", "pt-BR" ]; // P1 languages for /localweb page: Bangla, Hindi, Brazilian Portuguese
+  var otherLangs = [ "en-US", "da-DK", "sq", "sv", "es-CL", "fr", "ru", "fl", "es-MX" ]; // other languages that are at least 90% complete
+  var highlightOptions = [];
+  var options = [];
+  var lang;
+  var langData;
+  var config = window.angularConfig;
+
+  function createLangData(lang) {
+    return  {
+              id: lang,
+              title: config.langmap[lang] ? config.langmap[lang].nativeName : lang,
+              english: config.langmap[lang] && config.langmap[lang].englishName
+            };
+  }
+
+  for (var i = 0; i < config.supported_languages.length; i++) {
```

Comment:
Lets not do this. I think on production there won't be so many languages enable and it will be easy to find anyway. I tested it and it doesn't really do anything as it seems the value is being sorted again anyway.